docs/java/java-debugging.md

```diff
@@ -104,14 +121,21 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
 - `timeout` - Timeout value before reconnecting, in milliseconds (default to 30000ms).
 - `sourcePaths` - The extra source directories of the program. The debugger looks for source code from project settings by default. This option allows the debugger to look for source code in extra directories.
 - `projectName` - The preferred project in which the debugger searches for classes. There could be duplicated class names in different projects. This setting also works when the debugger looks for the specified main class when launching a program.
+- `stepFilters` - Skip specified classes or methods when stepping.
+  - `classNameFilters` - Skip the specified classes when stepping. Class names should be fully qualified. Wildcard is supported.
+  - `skipSynthetics` - Skip synthetic methods when stepping.
+  - `skipStaticInitializers` - Skip static initializer methods when stepping.
+  - `skipConstructors` - Skip constructor methods when stepping.
 
 ### User Settings
 
 - `java.debug.logLevel`: minimum level of debugger logs that are sent to VS Code, defaults to `warn`.
-- `java.debug.settings.showHex`: show numbers in hex format in **VARIABLES**, defaults to `false`.
-- `java.debug.settings.showStaticVariables`: show static variables in **VARIABLES**, defaults to `true`.
-- `java.debug.settings.showQualifiedNames`: show fully qualified class names in **VARIABLES**, defaults to `false`.
-- `java.debug.settings.maxStringLength`: the maximum length of string displayed in **VARIABLES** or **Debug Console**, the string longer than this length will be trimmed, defaults to `0` which means no trim is performed.
+- `java.debug.settings.showHex`: show numbers in hex format in "Variables" viewlet, defaults to `false`.
```

Comment:
Was this change in formatting intentional? The VS Code docs use bold for UI element titles that appear in the product and try to match the casing of the titles. We also prefer not to use "viewlet", and instead use "view", "section" or "pane" depending on the UI element.

Reply:
It's a copy-paste from other source, we can change it to the VS Code docs standard.

Reply:
I updated that section with my edit pass ;-)